package.json

```diff
@@ -38,13 +38,23 @@
     "db:shell:memeo": "docker exec -it aerogeardatasyncserver_postgres_memeo_1 psql -U postgresql -d memeolist_db"
   },
   "devDependencies": {
+    "apollo-cache-inmemory": "^1.3.0-beta.6",
     "apollo-fetch": "^0.7.0",
+    "apollo-link": "^1.2.2",
+    "apollo-link-http": "^1.5.4",
+    "apollo-link-ws": "^1.0.8",
+    "apollo-utilities": "^1.0.17",
     "ava": "1.0.0-beta.6",
+    "graphql-tag": "^2.9.2",
+    "node-fetch": "^2.2.0",
     "nodemon": "^1.17.5",
     "pre-commit": "^1.2.2",
-    "standard": "^11.0.1"
+    "standard": "^11.0.1",
+    "stoppable": "^1.0.6",
+    "ws": "^6.0.0"
   },
   "dependencies": {
+    "apollo-client": "^2.3.7",
```

Comment:
It is good that you used apollo-client. But, on the other hand, we have 2 clients now apollo-fetch and apollo-client.
I say we do nothing for now but we should migrate tests using apollo-fetch to apollo-client in the future.

Reply:
@aliok great point. I found this very easy to do so if you like, I can do it after your review (don't want to add more noise). 

Also, I'm just noticing that this should be a dev dependency :)

Reply:
> (don't want to add more noise).
Yeah, definitely.

> if you like, I can do it after your review 
even better :laughing: